starknet_devnet/util.py

```diff
@@ -263,3 +264,62 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 # FeederGatewayClient is implemented in such a way that it logs and raises;
 # this suppresses the logging
 suppress_feeder_gateway_client_logger = LogSuppressor("services.external_api.client")
+
+logger = logging.getLogger("gunicorn.error")
+
+
+class LogContext(dict):
+    """Context manager for logging transactions accumulating info while executing transaction"""
+
+    def get_context_name(self) -> str:
+        """Getting name of current context to put as first line in log entry"""
+        return getattr(self, "name", None) or "Transaction"
```

Comment:
Shouldn't it be `getattr(self, "name", "Transaction")`? Or even better, this class could have a property `name` 
 defined just before this method (initially set to `None`).

Reply:
removed cls